### ISSUE TITLE
infoWorldTimeFormatted showing incorrect day

### ISSUE THREAD

**Issue by Johndoedyou** [maruohon/minihud#9]
Formatted world time option displays a day over what is displayed on the F3 screen

**Comment by maruohon** [comment]
Yes, I guess it's debatable whether or not the first day in a world should be day 1 or day 0. Might depend on whether or not the player is a programmer... :p

Currently the mod specifically adds 1 to the value: https://github.com/maruohon/minihud/blob/master/src/main/java/fi/dy/masa/minihud/event/RenderEventHandler.java#L323

Should I add a config option just for this (whether or not to add that one)?

**Comment by Johndoedyou** [comment]
Yeah, I think giving an option to the player would be best, especially since it's debatable.

Nice mod by the way
